src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```
@@ -1,101 +1,39 @@
-.card {
-  border: 1px solid magenta; /** @TODO style these to spec, remove any prev imp styles necessary - https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6298%3A73041 */
-  position: relative;
-  display: block;
-  background: var(--white);
-  border-radius: 5px;
-  margin: 0;
-  padding: 20px;
-}
-
-/* visually hidden */
-.hidden {
-  clip: rect(0 0 0 0);
-  clip-path: inset(50%);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  white-space: nowrap;
-  width: 1px;
-}
-
 .linkbox {
-  composes: card;
   display: flex;
```

You may want to add `color` here so that the icon picks up the color as well. I think in the spec they are the same dark grey. 
```suggestion
  color: var(--token-color-foreground-primary);
  display: flex;
```